packages/core/examples/volumeEvents/index.ts

```diff
@@ -72,7 +72,7 @@ function updateLastEvents(number, eventName, detail) {
 
 let eventNumber = 1;
 
-const { IMAGE_RENDERED, CAMERA_MODIFIED, STACK_NEW_IMAGE } = Enums.Events;
```

Comment:
I see you removed STACK_NEW_IMAGE which makes sense, but could we then sub to VOLUME_NEW_IMAGE now and log that out in the last events?